doc/tb3_tutorial/README.md

```diff
@@ -0,0 +1,573 @@
+ - [Turtlebot Waffle](http://www.robotis.us/turtlebot-3-waffle-pi/)
+
+This tutorial may take about 2-5 hours to complete completely depends on your experience with ROS and robots and what computer system you have.
+
+## System Requirements:
+
+There is no minimum system requirement for this tutorial. A better system will help you build ROS2 packages faster and is going to deliver better performance when you run multiple tasks at the same time. This tutorial should work without any issues as long as you have a "newer" PC (less than 5 years old) with at least Intel i5 processor and 8 GB RAM. You will need some free disk space to install all the necessary software. I recommend you have at least 10 GB free disk space. 
+
+### Hardware/Operating System
+
+We tested this tutorial using the following systems:
+
+**System 1:**
+ - [NUC - NUCH8SV](https://www.intel.com/content/www/us/en/products/docs/boards-kits/nuc/mini-pcs/nuc-8-vr.html)
+	 - CPU: i7-8809G
+	 - GPU: Radeon RX Vega M GH graphics
+	 - 32 GB DDR4-2400 SO-DIMM RAM
+ - Operating System:
+	 - Ubuntu 18.04
+	 
+**System2:**
+- [ThinkPad T480](https://www.lenovo.com/us/en/laptops/thinkpad/thinkpad-t-series/ThinkPad-T480/p/22TP2TT4800?gclid=EAIaIQobChMIq9b80sqA4wIVCMNkCh1H5gG5EAAYASAAEgINKfD_BwE&cid=us:sem%7Cse%7Cgoogle%7C415787021890%7CThinkPad%20T480%7CIIP_NX_Thinkpad%20T%20Series_SMB%7C196131224&s_kwcid=AL!4030!3!343437426873!e!!g!!thinkpad%20t480&kw=thinkpad%20t480&adid=343437426873&addisttype=g&ef_id=EAIaIQobChMIq9b80sqA4wIVCMNkCh1H5gG5EAAYASAAEgINKfD_BwE:G:s&s_kwcid=AL!4030!3!343437426873!e!!g!!thinkpad%20t480)
+	- CPU i5 vPro (8gen) 1.7 Ghz
+	- 8 GB Ram
+- Operating System: Windows 10
+- Oracle VM VirtualBox
+- Ubuntu 18.04
+
+### Install Virtual Box (Optional)
+
+If you only want to try this tutorial out without chaining your PC's operating system, you can create a Virtual Machine and Install Ubuntu 18.04 on it. Just we aware of that if you pick this method building packages is going to take much longer time. You can download Virtual Box [here](https://www.virtualbox.org/).  If you have never done this before, there are a lot of good examples on the Internet. You can search them [here](https://www.google.com/search?q=Install%20Ubuntu%2018.04%20in%20Virtualbox).
+
+### Install Ubuntu 18.04
+
+![enter image description here](https://github.com/mlherd/ros2/blob/master/img/ubuntu_logo.jpeg?raw=true)
+######  Original Image Source:  https://www.ubuntufree.com/wp-content/uploads/2018/04/Ubuntu-bionic-beaver-official-logo-2018-e1524742194287-1024x473.jpeg
+
+Install the Desktop version of 18.04 
+
+ - http://releases.ubuntu.com/18.04/
+
+If you have never installed Ubuntu before, you can find the instructions here.
+
+ - [How to install Ubuntu?](http://releases.ubuntu.com/18.04/)
+
+# ROS2 Installation Steps
+
+After installing Ubuntu 18.04 and connecting it to the Internet, now it is time to install ROS2. You can do it by following the instructions below.
```

Comment:
Can we just point the user to the official ROS 2 install instructions? Otherwise we need to make sure we are always in sync with the upstream instructions

Reply:
In my opinion it is better if everything a user needs to complete the tutorial is in one page, but I agree with you. ROS 2 installation steps will probably change over time. I added a warning about it and added a link to the official ROS2 installation steps.